cityofaustin/atd-vz-data
# Add loading spinners to cards

## Associated issues
Closes https://github.com/cityofaustin/atd-data-tech/issues/16100

## Testing
**URL to test:** 
https://deploy-preview-1428--atd-vzv-staging.netlify.app/

**Steps to test:**
1. The spinners happen super fast sometimes so you may need to throttle your network to actually see anything. But just go to the summary page of the VZV and look for the new spinners while the summary cards are loading

Looking for design feedback so lmk what yall think. Heres a screenshot I took of when all the spinners are going so yall can see it easier:

![image](https://github.com/cityofaustin/atd-vz-data/assets/67130963/cb325d85-56b7-4d2e-97dc-205a3277d51c)

---
#### Ship list
